plugins/catalog-backend/migrationsv2/20210302150147_refresh_state.js

```diff
@@ -185,6 +185,11 @@ exports.up = async function up(knex) {
     table.index(['key'], 'search_key_idx');
     table.index(['value'], 'search_value_idx');
   });
+
+  // Delete bootstrap location which is no longer required.
+  await knex.schema.raw(
```

Comment:
This should not be in `schema` really. Besides, isn't it like

```ts
await knex('locations').where({ type: 'bootstrap', target: 'bootstrap' }).delete();
```

Reply:
This was some prior art from another migration that seems to work. But what you're suggesting also works so I changed it :)